### ISSUE TITLE
Code alpha internship task 3

### ISSUE THREAD

**Issue by Atif-baloch** [freeall/create-repository#1593]
from music21 import converter, instrument, note, chord
import glob

notes = []

for file in glob.glob("midi_songs/*.mid"):
    midi = converter.parse(file)
    parts = instrument.partitionByInstrument(midi)
    elements = parts.parts[0].recurse() if parts else midi.flat.notes

    for element in elements:
        if isinstance(element, note.Note):
            notes.append(str(element.pitch))
        elif isinstance(element, chord.Chord):
            notes.append('.'.join(str(n) for n in element.normalOrder))
import numpy as np

sequence_length = 100
pitchnames = sorted(set(notes))
note_to_int = dict((note, number) for number, note in enumerate(pitchnames))

network_input = []
network_output = []

for i in range(len(notes) - sequence_length):
    sequence_in = notes[i:i + sequence_length]
    sequence_out = notes[i + sequence_length]
    network_input.append([note_to_int[n] for n in sequence_in])
    network_output.append(note_to_int[sequence_out])

n_patterns = len(network_input)

X = np.reshape(network_input, (n_patterns, sequence_length, 1)) / float(len(pitchnames))
y = np.eye(len(pitchnames))[network_output]
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import LSTM, Dropout, Dense, Activation

model = Sequential()
model.add(LSTM(512, input_shape=(X.shape[1], X.shape[2]), return_sequences=True))
model.add(Dropout(0.3))
model.add(LSTM(512))
model.add
